index.html

```diff
@@ -8,60 +8,246 @@
-          <li id="twitter">
-            <a href="https://twitter.com/MercyMugambi15" target="_blank"
-              ><img src="./assets/icons/twitter.png"
-            /></a>
-          </li>
-          <li>
-            <a href="https://www.linkedin.com/in/mercymugambi/" target="_blank"
-              ><img src="./assets/icons/Linkedin.png"
-            /></a>
-          </li>
-          <li>
-            <a href="https://github.com/mercymugambi" target="_blank"
-              ><img src="/assets/icons/github.png"
-            /></a>
-          </li>
-          <li>
-            <a href="#medium" target="_blank"
-              ><img src="./assets/icons/medium.png"
-            /></a>
-          </li>
-          <li>
-            <a href="#angellist" target="_blank"
-              ><img src="./assets/icons/angellist.png"
-            /></a>
-          </li>
+      </div>
+    </section>
+    <!--about-->
+    <section class="about">
+      <div class="header1">
+        <h2 class="header">
+          About<br />
+          Myself
+        </h2>
+        <article class="heading-paragraph">
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+        </article>
+        <div class="heading-social">
+        <h2 class="icons-header">LET'S CONNECT</h2>
+        <div>
+          <ul class="icons" id="soc_icons">
+            <li>
+              <a href="#"
+                ><img src="./assets/icons/twitter.png" alt="twitter"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./assets/icons/Linkedin.png" alt="Linkedin"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./assets/icons/medium.png" alt="Medium"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./assets/icons/github.png" alt="Github"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./assets/icons/angellist.png" alt="Angellist"
+              /></a>
+            </li>
+          </ul>
+        </div>
+        <button type="button" id="popup">Get my resume</button>
+      </div>
+      </div>
+      <div id="about">
+        <ul class="lang">
+          <li><h4>Languages</h4></li>
+          <li><img src="./assets/images/arrow-down.png" alt="arrow.png" /></li>
+        </ul>
+        <ul class="icons-lang">
+          <li><img src="./assets/images/js.png" alt="js" />Javascript</li>
+          <li><img src="./assets/images/html.png" alt="html" />HTML</li>
+          <li><img src="./assets/images/css.png" alt="Ellipse" />CSS</li>
+        </ul>
+        <ul class="lang">
```

Comment:
As suggested by the previous reviewer, Kindly push your arrow a little bit left as shown in the comparison below
| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/219875578-cd73a93d-2ec0-4ad1-90db-efa854b0183b.png)| ![image](https://user-images.githubusercontent.com/61361037/219875559-ee48afb7-45b1-4620-9f7b-c329a903cab3.png)|